Add Starting Soon events

stash86 opened the pull request:
Fixes #39

stash86 committed:
Add Starting Soon events

silasary commented on gatherling/models/Event.php:
```diff
@@ -1027,6 +1027,26 @@ public static function getNextPreRegister($num = 20)
+    public static function getStartingSoon($playername)
+    {
+        $db = Database::getConnection();
+        $stmt = $db->prepare('SELECT e.name FROM events e, entries n WHERE n.event_id = e.id AND n.player = ? AND prereg_allowed = 1 AND active = 0 AND finalized = 0 AND DATE_SUB(start, INTERVAL 0 MINUTE) < NOW() ORDER BY start');
```
Why only prereg events?

stash86 replied:
Ah yeah. Forgot that some events might not allow prereg. Revising it now

stash86 replied:
On another note, the Your Upcoming Events will only show prereg events as well. Do we want to change it also? @silasary

stash86 committed:
Update Event.php

stash86 committed:
Include non-prereg events in Upcoming Events

Make sure events that doesn't allow prereg also shows up in Your Upcoming Events table

stash86 committed:
Apply fixes from StyleCI

silasary reviewed (approved):
Well.... it works.  For better or for worse.
![image](https://user-images.githubusercontent.com/194254/91765423-522ad680-ec1c-11ea-85c2-19240391f213.png)

mergify[bot] committed:
Merge branch 'dev' into starting-soon-event
